### PR TITLE
added skill to name founder of top tech companies

### PR DESCRIPTION
**Pull request by saurabhjn76**
Fixes issue #118 

Changes: added founder.txt in knowledge

Screenshots for the change: 
image
image

Skill can be tested at [http://dream.susi.ai/p/founder](http://dream.susi.ai/p/founder)

### CODE REVIEW

**Comment by saurabhjn76** [comment]
@DravitLochan @dynamitechetan @jyoth1raditya @AliAyub007 please review

**Comment by DravitLochan** [comment]
@saurabhjn76 Please provide a dream(etherpad) link
Thanks

**Comment by saurabhjn76** [comment]
@DravitLochan It's already there  [http://dream.susi.ai/p/founder](http://dream.susi.ai/p/founder) :+1:

File: models/general/knowledge/en/founder.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# tell about the founder of famous tech companies
+founder of *| who is the founder of * ?|Name the founder of * ?| Who is the father of * ?|can you tell me who is the founder of * ?| * is founded by whom ?
```
Comment (AliAyub007) [inline models/general/knowledge/en/founder.txt]: Add these questions as well. 
Who is founder of *
Name of founder of *
what is the name of founder of *
what is name of founder of *
Name of founder of *
Tell me about founder of *
just with star ( * founder )

**Comment by saurabhjn76** [comment]
@aliayub007 thanks, made changes.

**Review by hpdang — COMMENTED** [review]
@saurabhjn76 please make sure there's always at least 2 colleagues review your PR. Thanks

**Comment by saurabhjn76** [comment]
@hpdang :+1:

**Review by dynamitechetan — APPROVED** [review]

**Pull request merged**